commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/arguments/ListArgumentBuilder.java

```diff
@@ -161,7 +173,7 @@ private ListArgumentBuilderFinished(Function<T, IStringTooltip> mapper) {
 			 * @return a {@link ListArgument}
 			 */
 			public ListArgument<T> buildGreedy() {
-				return new ListArgument<>(nodeName, delimiter, allowDuplicates, supplier, mapper);
+				return new ListArgument<>(nodeName, delimiter, allowDuplicates, supplier, mapper, skipListValidation);
```

Comment:
This option should not only be used here. If it should be added, the `ListTextArgument` should also be able to work with this option.